docs/decisions/00005-api-decoupling-proxies

```diff
@@ -0,0 +1,51 @@
+##########################
+00005 API Decoupling Proxy
+##########################
+
+Context
+*******
+
+We are working on the first step of the `overall deprecation plan <https://discuss.openedx.org/t/deprecation-removal-ecommerce-service-depr-22/6839#roadmap-3>`__:
+- Decouple legacy ecommerce service from other systems by routing data through the Coordinator.
+- Integrate Coordinator with new back-end systems and migrate processes as piecemeal as we can.
+- Once nothing relies on the legacy integrations anymore, we can fully retire legacy ecommerce.
+
+This "decoupling" work is important for enabling smooth, risk-managed, migration to new backend services not only for edX.org's installation, but as a ramp for other Open edX deployments implementing their own Commerce Coordinator integrations as well.
+
+To start decoupling the legacy ecommerce service from other services and micro-frontends we’re building thin shims for the Coordiator and routing requests there and, for now, these shims will basically call the relevant legacy APIs.
+
+Decision
+********
+
+Use a proxy pattern when interposing Coordinator in front of other services' APIs, particularly while decoupling the legacy ecommerce service.
+
+Note: Commerce Coordinator does _not_ have to be in the middle of _every_ step; it should be used to facilitate coordination, especially _between_ systems involved in the order management workflow.  
+
```

Comment:
This note read a little open-ended to me at first, but I think it's just because the negative bit is first and the positive bit is second. How about reversing it like this: 
`Note: Commerce Coordinator should be used to facilitate coordination, especially _between_ systems involved in the order management workflow. It does _not_ have to be in the middle of _every_ step. `

Reply:
I like it!  Done.